x/qgb/keeper/keeper_valset.go

```diff
@@ -94,8 +94,15 @@ func (k Keeper) GetCurrentValset(ctx sdk.Context) (types.Valset, error) {
 
 		p := sdk.NewInt(k.StakingKeeper.GetLastValidatorPower(ctx, val))
 
-		// TODO make sure this  is always the case
-		bv := types.BridgeValidator{Power: p.Uint64(), EvmAddress: validator.EvmAddress}
+		evmAddress, exists := k.GetEVMAddress(ctx, val.String())
+		if !exists {
+			// validators may not have an EVM address associated with them.
+			// This is not enforced. We no longer include them from the validator
+			// set
```

Comment:
This changes the whole QGB security assumption, no? We rely on having all the validators signing, if not, then a Celestium becomes an ordinary DAC

Reply:
Just for posterity (and correct me @SweeXordious if you are under a different impression): the decision here is that we always want to have the validators committed to some address. If no address is provided then we derive the EVM address from the cosmos sdk validator address. Ideally we have a single CLI command that registers both in the same tx.